README.rst

```diff
@@ -19,6 +19,14 @@ Copy and paste the following steps to get started quickly on Ubuntu:
     sudo apt-get update
     sudo apt-get install -y mysql-client python-dev libmysqlclient-dev python-pip
     sudo pip install memsql
+    
+Copy and paste the following to get started with Amazon Linux: 
```

Comment:
can you update this to say `started with RHEL based distributions such as Amazon Linux or Centos`

Thanks!